src/OrbitGl/ThreadTrack.cpp

```diff
@@ -446,11 +446,17 @@ void ThreadTrack::OnCaptureComplete() {
 }
 
 [[nodiscard]] static inline uint64_t GetNextPixelBoundaryTimeNs(
-    float world_x, const internal::DrawData& draw_data) {
-  float normalized_x = (world_x - draw_data.track_start_x) / draw_data.track_width;
-  int pixel_x = static_cast<int>(
-      ceil(normalized_x * draw_data.viewport->WorldToScreenWidth(draw_data.track_width)));
-  return draw_data.min_tick + pixel_x * draw_data.ns_per_pixel;
+    uint64_t time_ns, const internal::DrawData& draw_data) {
```

Comment:
Could I ask for a more descriptive name for `time_ns`? XD

Reply:
I've renamed all the variables, let me know what you think.

Reply:
LGTM!